boards/posix/native_posix/Kconfig

```diff
@@ -26,4 +26,26 @@ config HAS_SDL
 	help
 	  This option specifies that the target board has SDL support
 
-endif
+if HAS_SDL
+
+config SDL_THREAD_INTERVAL
+	int "SDL thread sleep interval"
+	default 10
+	help
+	  Sleep interval time of SDL thread to handle events in microseconds.
```

Comment:
Beware that k_sleep only has the resolution of the system tick. Meaning not less than 10 milliseconds by default.
=> this definition seems a bit off